lib/fathead/caniuse/parse.py

```diff
@@ -31,15 +31,25 @@ def generate_answers(data):
         ])
         print ','.join(titles)
 
+        # Generate Related Topics
+        links = feature_data.get('links', [])
+        related = u' '.join([u'[[{} {}]]'.format(link['title'], link['url']) for link in links])
```

Comment:
Unfortunately I was wrong about the way Related Topics work. They can only be internal links to other Fathead articles so this isn't going to work.

Let's remove or comment this out for now 👍